pkg/hubble/filters/cluster_name.go

```diff
@@ -0,0 +1,65 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Hubble
+
+package filters
+
+import (
+	"context"
+	"fmt"
+
+	flowpb "github.com/cilium/cilium/api/v1/flow"
+	v1 "github.com/cilium/cilium/pkg/hubble/api/v1"
+)
+
+func sourceClusterName(ev *v1.Event) string {
+	return ev.GetFlow().GetSource().GetClusterName()
+}
+
+func destinationClusterName(ev *v1.Event) string {
+	return ev.GetFlow().GetDestination().GetClusterName()
+}
```

Comment:
minor nit, but I don't think these intermediate functions are necessary.

Reply:
Theses were "inspired" by the [`sourceIP`/`destinationIP`](https://github.com/cilium/cilium/blob/e49c7b3ddaf7e53a91a722ce7a1a328230749afc/pkg/hubble/filters/ip.go#L16-L22) but happy to change if there is a cleaner suggestion.